spec/system/admin/user_spec.rb

```diff
@@ -17,7 +45,10 @@
 
       let(:file) { 'users.csv' }
 
-      after { FileUtils.rm_f(file) }
+      after do
+        sleep(0.5) unless File.exist?(file)
+        FileUtils.rm_f(file) if File.exist?(file)
```

Comment:
This is fine for now, but eventually we could transfer the file downloading code to a helper file.